ogc-common/src/main/java/io/arlas/server/exceptions/OGC/OGCException.java

```diff
@@ -113,4 +116,10 @@ public Response getResponse() {
                 .type(MediaType.APPLICATION_XML).build();
         return response;
     }
+
+    public static OGCException getInternalServerException(Exception e, Service service, String locator) {
+        List<OGCExceptionMessage> ogcExceptionMessages = new ArrayList<>();
```

Comment:
You may use Arrays.asList(...) to make it more readable

Reply:
Fixed